src/Data/IdGenerator/Masa.Contrib.Data.IdGenerator.Snowflake.Distributed.Redis/DistributedWorkerProvider.cs

```diff
@@ -32,7 +32,7 @@ public class DistributedWorkerProvider : BaseRedis, IWorkerProvider
         ArgumentNullException.ThrowIfNull(distributedIdGeneratorOptions);
 
         _timestampType = distributedIdGeneratorOptions.TimestampType;
-        _recycleTime = distributedIdGeneratorOptions.RecycleTime;
+        _idleTimeOut = distributedIdGeneratorOptions.IdleTimeOut;
         _maxWorkerId = distributedIdGeneratorOptions.MaxWorkerId;
         _workerIdMinInterval = distributedIdGeneratorOptions.GetWorkerIdMinInterval;
         _currentWorkerKey = "snowflake.current.workerid";
```

Comment:
`Possible null reference assignment.` in 79 line.

Reply:
processed